notebooks/official/model_evaluation/custom_tabular_regression_model_evaluation.ipynb

```diff
@@ -686,10 +650,12 @@
       "source": [
         "#### Create task.py\n",
         "\n",
-        "In the next cell, you write the contents of the training script task.py.  In summary:\n",
+        "In the next cell, write the contents of the training script *task.py*.\n",
         "\n",
-        "- Get the directory where to save the model artifacts from the command line (`--model_dir`), and if not specified, then from the environment variable `AIP_MODEL_DIR`.\n",
-        "- Loads Boston Housing dataset from TF.Keras builtin datasets\n",
+        "To summarize, the script performs the following steps:\n",
+        "\n",
+        "- Gets the directory where to save the model artifacts from the command line (`--model_dir`), and if not specified, then from the environment variable `AIP_MODEL_DIR`.\n",
```

Comment:
Add "for":

- "Gets the directory for where to save the model artifacts from the command line..."